pages/common/peerflix.md

```diff
@@ -0,0 +1,15 @@
+# peerflix
+
+> Stream torrents to your favorite player.
+
+- Stream a torrent:
+
+`peerflix "magnet:?xt=urn:btih:ef330b39f4801d25b4245212e75a38634bfc856e"`
+
+- Stream a torrent to VLC:
+
+`peerflix "{{torrent url|magnet link}}" --vlc`
```

Comment:
Maybe the above example could mention (in the description) that it uses a magnet link, and this one could then be a (concrete) torrent URL, to demonstrate what format the command expects in this case. We can still include the `--vlc` here, since otherwise the syntax for this example is identical to the previous one.